pkg/api/validation/dynakube/eec.go

```diff
@@ -22,3 +23,17 @@ func extensionControllerImage(_ context.Context, _ *Validator, dk *dynakube.Dyna
 
 	return ""
 }
+
+func extensionControllerPVCStorageDevice(_ context.Context, _ *Validator, dk *dynakube.DynaKube) string {
+	if !dk.IsExtensionsEnabled() {
+		return ""
+	}
+
+	if dk.Spec.Templates.ExtensionExecutionController.UseEphemeralVolume != nil && *dk.Spec.Templates.ExtensionExecutionController.UseEphemeralVolume && dk.Spec.Templates.ExtensionExecutionController.PersistentVolumeClaim != nil {
```

Comment:
kind hard to read, could you make it more readable ? (like put it into a function?)